src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -314,10 +361,6 @@ private IAIState findTree()
                                  copy.getOrDefault(SAPLINGS_LIST, Collections.emptyList()),
                                  worker.getCitizenColonyHandler().getColony());
             }
-
-            // Delay between area searches
-            setDelay(100);
-            return getState();
```

Comment:
can you please make sure this isn't being run every tick or whatever now?

Reply:
I think this is a fragment of older code. These lines will now fall through to line 377 (the return), which will change the state to LUMBERJACK_NO_TREES_FOUND, which has a delay built-in.